dcm4che-core/src/main/java/org/dcm4che3/io/DicomInputStream.java

```diff
@@ -831,10 +832,16 @@ private void checkIsThis(DicomInputStream dis) {
             throw new IllegalArgumentException("dis != this");
     }
 
-    private void skipAttribute(String message) throws IOException {
-        LOG.warn(message,
-                 new Object[] { TagUtils.toString(tag), length, tagPos });
-        skipFully(length);
+    /**
+     * @param message the message to use in the warning log message
+     * @param methodName the name of the method that is skipping the attribute
+     * @throws IOException potentially thrown when performing the 'skip' operation
+     */
+    private void skipAttribute(String message, String methodName) throws IOException {
+        String tagAsString = TagUtils.toString(this.tag);
+        LOG.warn(message, tagAsString, length, tagPos, methodName);
+        long skipLength = skip(length);
+        LOG.debug("Skipped {} actual bytes for {}", skipLength, tagAsString);
```

Comment:
I'll out my comment from https://github.com/dcm4che/dcm4che/pull/1152 again here:

Hmm, I wonder if this call to skip(.) here shouldn't actually be a call to skipFully(.)? (@gunterze)

Reply:
Yes, replacing skipFully() by skip() actually changes the behavior!

Reply:
Ah, yes indeed! I didn't notice that on master there was already skipFully here, because on the generic-config branch there was also a skip() here before. On the generic-config branch, the fix for this (https://github.com/dcm4che/dcm4che/pull/404) was never applied!

@craigtmoore: instead of now breaking it on master again, I would suggest to actually port the fix back to your generic-config branch.

Reply:
OK, I've changed it back to skipFully(). Good catch!